modules/tensor_mechanics/src/materials/SingleVariableReturnMappingSolution.C

```diff
@@ -350,7 +351,7 @@ bool
 SingleVariableReturnMappingSolution::converged(const Real residual, const Real reference)
 {
   return (std::abs(residual) <= _absolute_tolerance ||
-          (std::abs(residual) / reference) <= _relative_tolerance);
+          std::abs(residual / reference) <= _relative_tolerance);
```

Comment:
Was this change needed?  Seems like it's about the same since reference is always positive.

Reply:
That is true in your current implementations, but there are cases where reference can be negative if the scalar increment is negative, or there is a different reference residual than currently implemented. I could protect this using `std::abs()` in `computeReferenceResidual`, but it seems easier just to move this parentheses and always ensure the reference residual is being calculated.